MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,572 @@
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+ImageEditorToolbarItem shapeItem = (ImageEditorToolbarItem)footerToolbar.ToolbarItems.FirstOrDefault(i => i.Name == "Shape");
+ImageEditorToolbar subtoolbar = shapeItem.SubToolbars[0];
+ImageEditorToolbarItem arrowItem = (ImageEditorToolbarItem)subtoolbar.ToolbarItems.FirstOrDefault(i => i.Name == "Arrow");
+
+{% endhighlight %}
+
+N> You can remove the existing toolbar items `Name` from image editor toolbarItems collection based on the index value and change the icon and text values dynamically for any of the already added toolbar item based on the index.
+
+## To change item visibility
+
+You can hide or show the toolbar items by specifying their icon names and setting the boolean values to true or false.
+
+N> You can customize an icon by specifying its `Name`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+        SfImageEditor imageEditor = new SfImageEditor();
+        imageEditor.Source = ImageSource.FromFile("image.png");
+        imageEditor.SetToolbarItemVisibility("Shape", false);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## To change item enable state
+
+You can enable or disable the toolbar items by using `IsEnabled` property in `ImageEditorToolbarItem` class. Specifying their icon names and setting the boolean values to true or false.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem browseGroup = (ImageEditorToolbarGroupItem) headerToolbar.ToolbarItems[0];
+ImageEditorToolbarItem browseItem = browseGroup.Items.FirstOrDefault();
+browseItem.IsEnabled = false;
+
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem) headerToolbar.ToolbarItems[2];
+ImageEditorToolbarItem saveItem = saveGroup.Items.FirstOrDefault(i => i.Name == "Save");
+saveItem.IsEnabled = false;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## To change item view
+
+To Customize the toolbar items view by specifying their icon names and setting the view using `View` property in `ImageEditorToolbarItem`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+Image image = new Image();
+image.Source = ImageSource.FromResource("MyProject.Resources.Images.zoom.jpg");
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem zoomGroup = (ImageEditorToolbarGroupItem)headerToolbar.ToolbarItems[1];
+ImageEditorToolbarItem zoomItem = (zoomGroup).Items.FirstOrDefault(i => i.Name == "ZoomIn");
+zoomItem.View = image;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add a new sub toolbar by clearing existing sub toolbar 
+
+You can create `SubToolbars` within your toolbar items to organize related actions.
+
+{% tabs %}
+
+{% highlight XAML %}
+
+<imageEditor:SfImageEditor Source="image.png"
```

Comment:
This example shows new toolbar creation. Add example for creating new sub toolbar for existing item.

Get crop item and add new sub toolbar for it

Reply:
Updated